src/components/landing/Hero.tsx

```diff
@@ -17,33 +17,24 @@ export default function Hero() {
   return (
     <Wrapper>
       <HeroTitle>
-        Your gateway to <span>Arweave and AO</span>
+        ArConnect is now <span>Wander</span>
       </HeroTitle>
       <Spacer y={1.55} />
       <HeroDescription>
         A non-custodial Arweave and AO native wallet with extensive features
         <br />
         all in your favorite browser and mobile device
       </HeroDescription>
-      <Spacer y={2.5} />
-      <Buttons>
-        <Link href="/download" passHref legacyBehavior>
-          <Button onClick={() => sendConversion("tw-ofnyd-og7tu", {})}>
-            Download
-            <ArrowUpRightIcon />
-          </Button>
-        </Link>
         <Button
```

Comment:
```suggestion
      <Buttons>
        <Button
```
You inadvertedly removed an HTML Tag opener and it breaks the doc